libs/shared/console-shared/src/lib/cluster-settings/ui/cluster-resources-settings/cluster-resources-settings.tsx

```diff
@@ -133,7 +138,7 @@ export function ClusterResourcesSettings(props: ClusterResourcesSettingsProps) {
               required: 'Please select a cluster type',
             }}
             render={({ field }) => (
-              <div>
+              <div className="pointer-events-none">
```

Comment:
@RemiBonnet Why do we need that?

Reply:
Sorry, I forgot to add a comment for this. Initially, we had two radio buttons, but now there’s only one for all cloud providers. This might cause a bug with Karpenter. I added this quickly to fix it because the UI has changed, we need to revamp it soon!

https://github.com/user-attachments/assets/13d891e3-945f-491a-8e49-a1271fa4b147

Reply:
Gotcha! Thx for clarifying 😉